src/routes/docs/sdks/+page.markdoc

```diff
@@ -71,6 +71,12 @@ Server libraries for integrating with Appwrite to build server side integrations
 * [appwrite/sdk-for-dart](https://github.com/appwrite/sdk-for-dart)
 * 
 ---
+* {% only_dark %}{% icon_image src="/images/platforms/dark/go.svg" alt="Go logo" size="m" /%}{% /only_dark %}
+{% only_light %}{% icon_image src="/images/platforms/go.svg" alt="Go logo" size="m" /%}{% /only_light %}
+* Go SDK `1.22`
+* [appwrite/sdk-for-go](https://github.com/appwrite/sdk-for-go)
```

Comment:
@TorstenDittmann, @Meldiron, should the go sdk be marked as beta? react native and .net came before and those are still beta.

Reply:
Torsten said to mark this as beta